include/ignition/gui/qml/GzPose.qml

```diff
@@ -72,10 +78,14 @@ Item {
   // Expand/Collapse of this widget
   property bool expand: true
 
+  property var gzPlotMimeData
 
   /*** The following are private variables: ***/
   height: gzPoseContent.height
 
+  property int iconWidth: 20
+  property int iconHeight: 20
```

Comment:
How about keeping these inside `GzPlotIcon`? This way we ensure it's always consistent across widgets